meta/meta.go

```diff
@@ -147,7 +155,14 @@ func (m *Meta) GenGlobalID() (int64, error) {
 	globalIDMutex.Lock()
 	defer globalIDMutex.Unlock()
 
-	return m.txn.Inc(mNextGlobalIDKey, 1)
+	newID, err := m.txn.Inc(mNextGlobalIDKey, 1)
+	if err != nil {
+		return 0, errors.Trace(err)
+	}
+	if newID > MaxGlobalID {
```

Comment:
This possible function is currently called in several places, and whether adding this restriction will affect it

Reply:
no, it is a huge value. if we allocate 100k/s we can use it for nearly 90 years, so I think no one can achieve it. And we also have a check on the bootstrap stage